api/v1beta1/ssp_webhook.go

```diff
@@ -159,3 +167,16 @@ func validateOperandPlacement(namespace string, placement *api.NodePlacement) er
 
 	return clt.Create(context.TODO(), deployment, &client.CreateOptions{DryRun: []string{metav1.DryRunAll}})
 }
+
+// TODO: also validate DataImportCronTemplates in general once CDI exposes its own validation
+func validateDataImportCronTemplates(r *SSP) error {
+	for _, cron := range r.Spec.DataImportCronTemplates {
+		if len(cron.Name) < 1 {
+			return fmt.Errorf("missing name in DataImportCronTemplate")
+		}
+		if len(cron.Namespace) > 0 && cron.Namespace != GoldenImagesNSname {
```

Comment:
This may be a good time to allow deploying golden images to any namespace, and not hard-code a constant namespace.
Like we do for VM templates.

Reply:
I agree, but would prefer to have that as a separate decision and not as part of this API change.